dev-python/pgspecial/pgspecial-1.11.10.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8} )
+inherit distutils-r1
+
+DESCRIPTION="Python implementation of PostgreSQL meta commands"
+HOMEPAGE="https://github.com/dbcli/pgspecial"
+SRC_URI="https://github.com/dbcli/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
Don't use `${PN}` in SRC_URI :)
(https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide/Style_Guide#Plaintexting_.24.7BPN.7D)